### PR TITLE
Tina CMS - Improve Collections

### LINKED ISSUE #1522: 💸 TinaCMS - Improve COLLECTIONS titles
cc @bradystroud @jaydenalchin @adamcogan 

Hi @Harry-Ross 

I think we can make the sections' (collections) titles more friendly/consistent in TinaCMS

- [x] 1. Change from "Employment Pages" to "Employment - Index"
- [x] 2. Change from "Opportunities" to "Employment - Opportunities"
- [x] 3. Change from "Technology Cards" to "Consulting -  Technology Cards"
- [x] 4. Change from "Technology Badges" to "Consulting -  Technology Badges"
- [x] 5. Change from "Office - Index" to "Offices - Index"
- [x] 5. Change from "Office Pages" to "Offices Pages"
- [x] 6. Move Offices sections to be together (Make index the 1st)
- [x] 7. Change from "Presenters" to "Events - Presenters"
- [x] 8. Change from "Locations" to "Events - Locations"
- [x] 9. Move Events sections to be together (Make index the 1st)
- [x] 10. Change from "Marketing" to "Marketing Pages"
- [x] 11. Change from "Payment Details" to "Company - Payment Details"
- [x] 12. Move Company sections to be together (Make index the 1st)
- [x] 13. Change from "Testimonial Categories" to "Testimonials - Categories"
- [ ] 14. Why is "newsletters.mdx" under "Pages" and not "Newsletters"? Should we have a new section "Newsletters - Index" to put that file there?

---

- [x] Once all the above is done please send a test please to @tiagov8 

Screenshot 2023-10-10 at 9 20 59 AM

### PR DESCRIPTION
**Pull request by amankumarrr**
- Affected routes: `/*`

- Fixed #1522 

Reordered collection in logical order

- [ ] Include done video or screenshots

image
<br>
image

**Figure: Collections order by area**

### CODE REVIEW

File: services/server/events.ts [hunk 1]
```diff
@@ -43,7 +43,11 @@ export const getEvents = async (odataFilter: string): Promise<EventInfo[]> => {
     );
     return [];
   }
-
+  process.env.MICROSOFT_OAUTH_CLIENT_ID,
+    console.log(
+      "🚀 ~ file: events.ts:47 ~ getEvents ~ process.env.MICROSOFT_OAUTH_CLIENT_ID,:",
```
Comment (bradystroud) [inline services/server/events.ts]: I would add an explanation here

"Missing token"

**Review by bradystroud — CHANGES_REQUESTED** [review]
Hey @amankumarrr 
This is cool, but I think TinaCMS could really improve this list by adding a simple search.

Can you also create an issue there and also send it in Discord?

File: services/server/events.ts (hunk 1 above)
Comment (amankumarrr) [reply services/server/events.ts]: That was an accidental commit, reverted it in new commits.

**Review by bradystroud — APPROVED** [review]

**Comment by tiagov8** [comment]
cc @jaydenalchin 

Hi @amankumarrr 

I have a few more changes:

1. Can you change from "Global" to "Global - Sections"
2. Change from "Pages" to "Root - Pages" OR "Base - Pages" (discuss with @bradystroud and others what is better)
3. Change from "Products" to "Products - Pages"
4. Add a dash before every collection for "Pages". E.g. "Consulting - Pages"
5. Make "Root/Base - Pages" the 1st item
6. Make "Global - Sections" the 2nd item
7. Make all the rest in alphabetical order
8. Tell me to review again

**Comment by tiagov8** [comment]
Test pass ✅
Looking much better. Thanks @amankumarrr

**Pull request merged**